bin/vulnxml2json5.py

```diff
@@ -0,0 +1,192 @@
+#! /usr/bin/python3
+#
+# Convert our XML file to a JSON file as accepted by Mitre for CNA purposes
+# as per https://github.com/CVEProject/automation-working-group/blob/master/cve_json_schema/DRAFT-JSON-file-format-v4.md
+#
+# ASF httpd and OpenSSL use quite similar files, so this script is designed to work with either
```

Comment:
This (or associated files) doesn't belong here any longer, does it?  I assume someone is going to use it once in the OMCDATA repository to convert vulnerabilities.xml to the corresponding JSON, so it probably belongs in that repo rather than here.

Reply:
It was a one-off script that we most likely will never need to use again; suggestions for proper location (and for a copy of the legacy vulnerabilities.xml)?

Reply:
Together with vulnerabilities.xml, i.e. in the OMCDATA repository

Reply:
(then we can drop them all in one PR later on)